lms/envs/test.py

```diff
@@ -368,6 +368,8 @@
     'django.core.files.uploadhandler.TemporaryFileUploadHandler',
 ]
 
+BLOCK_STRUCTURES_SETTINGS['PRUNING_ACTIVE'] = True
```

Comment:
We enable this for tests because it's used as a boolean "this feature is enable" flag - without it on, `_prune_files` is a very-hard-to-debug no-op.